mods/voidmulti.sc2mod/base.sc2data/gamedata/abildata.xml

```diff
@@ -119,6 +122,11 @@
         <CmdButtonArray index="Execute">
             <Flags index="ShowInGlossary" value="0"/>
         </CmdButtonArray>
+        <InfoArray index="0">
+            <SectionArray index="Stats">
+                <DurationArray index="Duration" value="0"/>
```

Comment:
**undocumented**: Drone's stats is now changed immediately instead of after 0.4443
Does this cause unburrowing units to become "visible" immediately instead of after ~300ms?